src/main/java/com/sapred/ordermanagerred/controller/UserController.java

```diff
@@ -18,27 +24,48 @@ public class UserController {
 
     @GetMapping("/logIn/{email}/{password}")
     public ResponseEntity<String> logIn(@PathVariable("email") String email, @PathVariable("password") String password) {
-        return userService.logIn(email, password);
+        try {
+            return new ResponseEntity<String>(userService.logIn(email, password), HttpStatus.OK);
+        } catch (ResponseStatusException e) {
+            return new ResponseEntity<String>(e.getMessage(), e.getStatusCode());
+        }
     }
 
-    // זה פונקציה שמכניסה נתונים רק בשביל לבדוק שההתחברות עובדת
     @GetMapping("/fill")
     public void fill() {
         userService.fill();
     }
 
-    @DeleteMapping("/deleteCustomer/{customerId}")
-    public ResponseEntity<Boolean> deleteCustomer(@RequestHeader("token") String token, @PathVariable("customerId") String customerId) {
-        return userService.deleteCustomer(token, customerId);
+    @DeleteMapping("/{userId}")
+    public ResponseEntity<Boolean> deleteUser(@RequestHeader("token") String token, @PathVariable("userId") String userId) {
+        try {
+            userService.deleteUser(token, userId);
+            return new ResponseEntity<>(true, HttpStatus.OK);
+        } catch (NoPermissionException e) {
+            return new ResponseEntity<>(false, HttpStatus.UNAUTHORIZED);
+        } catch (RuntimeException e) {
```

Comment:
RuntimeException  is extends from Exception so you can catch only Exception and throw 500

Reply:
Please do